README.md

```diff
@@ -23,7 +23,8 @@ A quickstart page can be found under [https://aliceo2group.github.io/](https://a
 In order to build and install O2 with aliBuild you can follow [this tutorial](http://alisw.github.io/alibuild/o2-tutorial.html).
 
 ### Issue tracking system
-We use JIRA to track issues. [Report a bug here](https://alice.its.cern.ch/jira/secure/CreateIssue.jspa?pid=11201&issuetype=1).
+We use JIRA to track issues. [Report a bug here](https://alice.its.cern.ch/jira/secure/CreateIssue.jspa?pid=11201&issuetype=1). 
+Add the JIRA issue key (e.g. `O2-XYZ`) to the PR title or in a commit message to have the PR/commit appear in the JIRA ticket. 
```

Comment:
nice! didn't know that.

Reply:
yes, it's nice, albeit the link is a bit hidden. We should also have some bot which goes around the issues and substitutes `O2-XYZ` with an actual link to our JIRA.

Reply:
Actually it looks like it's only available for the JIRA Cloud.

https://github.com/integrations/jira

Reply:
@matthiasrichter 👍 

@ktf yes, the bot exists but only for JIRA Cloud 😦   I have already told Atlassian what I think of it. 

So we have a one way integration that works well (GH->JIRA) and the other that is inexistant. As you mention, it is a simple bot to replace `O2-XYZ` into the full URL. It should not be complicated. Maybe a service task ?

Reply:
Might as well, albeit I wouldn't say it's urgent.